client/state/themes/selectors.js

```diff
@@ -325,12 +325,18 @@ export function isWporgTheme( state, themeId ) {
 /**
  * Whether a theme is present on WordPress.com.
  *
+ * Returns false if themeId is installed on (jetpack) siteId, since
+ * installed themes take priority. For example, this can be the case
+ * with themes available on both wpcom and .org such as 'twentysixteen'.
+ *
  * @param  {Object}  state   Global state tree
  * @param  {Number}  themeId Theme ID
+ * @param  {Number}  siteId  Site ID
  * @return {Boolean}         Whether theme available on WordPress.com
  */
-export function isWpcomTheme( state, themeId ) {
-	return !! getTheme( state, 'wpcom', themeId );
+export function isWpcomTheme( state, themeId, siteId ) {
```

Comment:
Agreed (and noted [here](https://github.com/Automattic/wp-calypso/pull/11087#issuecomment-276999024) ). I just don't have a good feel for what the name should be at the moment—I think it will become clear after #10972 and changing the filtering. Suggestions welcome.